File: Ejercicios Propuestos 2/Ejercicio 9/ejercicio9.cpp
#include <iostream>

using namespace std;

int main() {
    // Definimos las variables que utilizaremos para nuestras tablas    
    int inicio, fin, tope, i, j, resultado;    
   
    cout << "¿Desde qué tabla desea iniciar?" << endl;
    cin >> inicio;
    
    cout << "¿Hasta qué tabla desea llegar?" << endl;
    cin >> fin; 
    
    // límite para la multiplicación
    cout << "¿Hasta qué número desea multiplicar en cada tabla?" << endl;
    cin >> tope;
    
    cout << "--------------------------------------------------" << endl;
    
    // generar y mostrar nuestras tablas
    for (i = inicio; i <= fin; i++) {
        cout << "=========================" << endl;
        cout << "      TABLA DEL " << i << endl;
        cout << "========================" << endl;
        
        // Realizamos el proceso de las multiplicaciones iterando hasta el tope
        for (j = 1; j <= tope; j++) {
            resultado = i * j;
            cout << i << " x " << j << " = " << resultado << endl;
        }
        
        cout << endl; 
    }
    
    return 0;
}

File: Ejercicios Propuestos 2/Ejercicio 9/Ejercicio 9.jl
# Rangos de las tablas:
println("¿Desde qué tabla desea iniciar?")
inicio = parse(Int, readline())

println("¿Hasta qué tabla desea llegar?")
final = parse(Int, readline())

# Límite de la multiplicación
println("¿Hasta qué número desea multiplicar en cada tabla?")
tope = parse(Int, readline())

println("--------------------------------------------------")

# Proceso de generar las tablas 
for i in inicio:final
    println("=========================")
    println("      TABLA DEL ", i)
    println("========================")
    
    # Proceso de las multiplicaciones
    for j in 1:tope
        global resultado
        resultado = i * j
        println(i, " x ", j, " = ", resultado)
    end
    
    println("") 
end